salt/modules/drbd.py

```diff
@@ -1,13 +1,47 @@
 # -*- coding: utf-8 -*-
 '''
-DRBD administration module
+Module to provide DRBD functionality to Salt
+
+.. versionadded:: pending
+
+:maintainer:    Nick Wang <nwang@suse.com>
+:maturity:      alpha
+:depends:       ``drbdadm`` drbd utils
+:platform:      all
+
+:configuration: This module requires drbd kernel module and drbd utils tool
+
+.. code-block:: yaml
+
 '''
 from __future__ import absolute_import, print_function, unicode_literals
 
 import logging
+
+from salt.exceptions import CommandExecutionError
 from salt.ext import six
 
-log = logging.getLogger(__name__)
+import salt.utils.json
+import salt.utils.path
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
is there a reason you are changing this to `LOGGER` instead of `log`. It looks like in salt's code the convention is `log = loggin..`

Reply:
Will follow the salt style, change to `log`